components/core/src/clp/clo/OutputHandler.hpp

```diff
@@ -156,15 +174,17 @@ class ResultsCacheOutputHandler : public OutputHandler {
     // Methods inherited from OutputHandler
     /**
      * Adds a result to the batch.
-     * @param original_path
-     * @param message
-     * @param timestamp
+     * @param orig_file_path original path of the file that input log event belongs to
```

Comment:
If we're deleting the CountOutputHandler docstring, then we should delete this one too.